### DISCUSSION TITLE
LearnOqtane.com LEARN | BUILD | DEPLOY | MONETIZE

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3172] [Show and tell]
I plan to launch [LearnOqtane.com](https://learnoqtane.com) which will have video courses and video tutorials on how to develop themes and modules on this great web application framework.

Please check it out and subscribe to the mailing list so that you can be notified when the site launches.

The site is dog fooding the dev branch of Oqtane.

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/ca5b05e2-85d7-4c18-9e6b-4c9fdca856a9)

**Comment by sbwalker** [comment]
@vnetonline the correct spelling of "monitize" is "monetize"

**Reply by vnetonline** [reply]
Thanks for the pickup

**Comment by vnetonline** [comment]
The spelling of Monetize has been fixed 

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/d8b69a8c-113a-4394-8db9-5e6c07e7d1d6)

**Comment by vnetonline** [comment]
I have created a project and org for LearnOqtane.com where you can track the progress of all things which I am working on in order to launch LearnOqtane.com

[https://github.com/orgs/LearnOqtane/projects/1](https://github.com/orgs/LearnOqtane/projects/1)

if you have any Courses or Lessons you would like me to cover you can add them to this project

Thanking you for your help in advance
